index.md

```diff
@@ -4,6 +4,11 @@ site: sandpaper::sandpaper_site
 
 This is a new lesson built with [The Carpentries Workbench][workbench]. 
 
+## Background
+
+This lesson was initially created at 
+[Collaborations Workshop '24](https://www.software.ac.uk/workshop/collaborations-workshop-2024-cw24), 
+an event hosted by the [Software Sustainability Institute (SSI)](https://www.software.ac.uk/).
 
 [workbench]: https://carpentries.github.io/sandpaper-docs
```

Comment:
This was a reference from the text that you have deleted. I will delete it myself in the next cleanup.